src/pudl_zenodo_storage/zs/metadata.py

```diff
@@ -74,10 +74,8 @@ def _generate_metadata(data_source_id, data_source_uuid):
 epacems = _generate_metadata("epacems", epacems_uuid)
 
 # Unaltered EPACEMS-EIA Crosswalk archive.
-epacems_unitid_eia_plant_crosswalk_uuid = "40696588-d6ee-11ec-abb9-34363bce6e4c"
-epacems_unitid_eia_plant_crosswalk = _generate_metadata(
-    "epacems_unitid_eia_plant_crosswalk", epacems_unitid_eia_plant_crosswalk_uuid
-)
+epacamd_eia_uuid = "3bad6051-a0c3-43a2-97a6-02f61460ab35"
+epacamd_eia = _generate_metadata("epacamd_eia", epacamd_eia_uuid)
```

Comment:
I think this should be the same as the old UUID, not a new UUID -- we aren't creating a new archive lineage are we? It'll just continue to use the existing one, but with a new name.

Reply:
I guess technically yes! I'll change it back

Reply:
I changed it back